.idea/compiler.xml

```diff
@@ -2,6 +2,7 @@
 <project version="4">
   <component name="CompilerConfiguration">
     <annotationProcessing>
+      <profile default="true" name="Default" enabled="true" />
```

Comment:
So I think if we want to git ignore this, we might want to git ignore the entirety of both .idea and .vscode, since both of those folders are also just IDE settings. The pro would be cleaning up the codebase and commit history, while the con would be new users needing to manually set up IDE configurations. Any preference on this? Either way is fine by me :)